doc/tutorial/single_cell_recipe.rst

```diff
@@ -0,0 +1,181 @@
+
+Step **(4.2)** defines that this model has one cell.
+
+Step **(4.3)** defines that this model has one source.
+
+Step **(4.4)** returns the :class:`arbor.cell_kind.cable`. Make sure you don't instantiate the model with a non cable cell! Since we have just one cell, we can ignore :gen:`gid`.
+
+Step **(4.5)** returns the cell description passed in on class initialisation. If you would have multiple cells of different kinds, make sure the right types are returned when :meth:`arbor.recipe.cell_kind` and :meth:`arbor.recipe.cell_description` are called with the same :gen:`gid`.
+
+Step **(4.6)** returns the probes passed in at class initialisation.
+
+Step **(4.7)** returns the properties that will be applied to all cells of that kind in the model.
+
+More methods can be overridden if your model requires that, see :class:`arbor.recipe` for options.
+
+Step **(5)** instantiates the recipe with the cable cell described earlier, and a single voltage probe located at "center".
+
+The context and domain decomposition
+------------------------------------
+
+:class:`arbor.single_cell_model` does not only take care of the recipe, it also takes care of defining how the simulation will be ran. When you create and use your own recipe, you'll need to do this manually, in the form of defining a hardware context and a domain decomposition. Fortunately, the default constructors of :class:`arbor.context` and :class:`arbor.partition_load_balance` are sufficient for this model, and is what :class:`arbor.single_cell_model` does under the hood! We'll leave the details of this subject for another tutorial.
+
+.. code-block:: python
+
+    # (6) Create a default execution context and a default domain decomposition.
+
+    context = arbor.context()
+    domains = arbor.partition_load_balance(recipe, context)
+
+Step **(6)** sets up a default context and domains.
+
+The simulation
+--------------
+
+.. code-block:: python
+
+    # (7) Create and run simulation and set up 10 kHz (every 0.1 ms) sampling on the probe.
+    # The probe is located on cell 0, and is the 0th probe on that cell, thus has probe_id (0, 0).
+
+    sim = arbor.simulation(recipe, domains, context)
+    sim.record(arbor.spike_recording.all)
+    handle = sim.sample((0, 0), arbor.regular_schedule(0.1))
+    sim.run(tfinal=30)
+
+Step **(7)** instantiates the simulation and sets up the probe add in step 6. In the :class:`arbor.single_cell_model` version of the model the probe was set to record at a frequency of 10 kHz, and a simulation duration of 30 ms. Note that spike recording must be switched on. For extraction of the probe traces later on, we store a handle.
```

Comment:
- Split over several lines.
-`sets up the probe add*ed*`
-`version of this *example*`
-ambiguous: `the probe was set to record at a frequency of 10 kHz, and a simulation duration of 30 ms` -> `the probe frequency and simulation duration are the same.`